libpod/runtime.go

```diff
@@ -1469,6 +1469,24 @@ func (r *Runtime) GetOCIRuntimePath() string {
 // TODO Once runc has support for cgroups, this function should be removed.
 func cgroupV2Check(configPath string, tmpConfig *RuntimeConfig) error {
 	if !tmpConfig.CgroupCheck && rootless.IsRootless() {
+		if tmpConfig.CgroupManager == SystemdCgroupsManager {
+			// If we are running rootless and the systemd manager is requested, be sure that dbus is accessible
+			session := os.Getenv("DBUS_SESSION_BUS_ADDRESS")
+			hasSession := session != ""
+			if hasSession && strings.HasPrefix(session, "unix:path=") {
```

Comment:
Under what circumstances will `session` _not_ begin with `unix:path=`? Is it OK to leave `hasSession` as true in those cases?

Reply:
Then it would not a unix domain socket,  I guess this would be a stange setup/

Reply:
Well, in my netsearching I see there's the possibility of DBUS_etc being `unix:abstract=` ; and it doesn't seem uncommon. I can't find a spec defining the exact definition of `DBUS_etc`, and don't really want to spend my entire day netsearching. I do think it's important that if we're mucking with / testing for magic values in `DBUS_etc` we should have a thorough understanding of what we're dealing with.

Reply:
if we are not able to handle it, it is safer to leave it to "true" instead of blocking it too early.

If the user session is not valid anyway, the container creation will fail later.